src/common/hdfs/HdfsHelper.h

```diff
@@ -26,6 +26,15 @@ class HdfsHelper {
                                               const std::string& hdfsPath,
                                               const std::string& localPath) = 0;
 
+    virtual StatusOr<bool> exist(const std::string& hdfsHost,
```

Comment:
So you don't implement this function?

Reply:
Implement it in another class.

Reply:
why don't define it as a pure virtual function ?